docs/README.md

```diff
@@ -1,10 +1,59 @@
 [Return to Project Root](../)
 
 # User Documentation
-tbd
+sail-operator manages the lifecycle of your Istio control planes and aims to become a vehicle for automated integration of Istio with compatible third-party software. Instead of creating a new configuration schema, sail-operator APIs are built around Istio's helm chart APIs. All installation and configuration options that are exposed by Istio's helm charts are available through the sail-operator CRDs' `values` fields.
 
 ## Concepts
-tbd
+
+### Istio resource
+The `Istio` resource is used to manage your Istio control planes. It is a cluster-wide resource, as the Istio control plane operates in and requires access to the entire cluster. To select a namespace to run the control plane pods in, you can use the `spec.namespace` field. You can access all helm chart options through the `values` field in the `spec`:
+
+```yaml
+apiVersion: operator.istio.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.22.3
+  namespace: istio-system
+  updateStrategy:
+    type: InPlace
+  values:
+    pilot:
+      resources:
+        requests:
+          cpu: 100m
+          memory: 1024Mi
+```
+
+To support canary updates of the control plane, sail-operator includes support for multiple Istio versions. You can select a version by setting the `version` field in the `spec` to the version you would like to install, prefixed with a `v`. You can then update to a new version just by changing this field.
+
+sail-operator supports two different update strategies for your control planes: `InPlace` and `RevisionBased`. When using `InPlace`, the operator will immediately replace your existing control plane resources with the ones for the new version, whereas `RevisionBased` uses Istio's canary update mechanism by creating a second control plane to which you can migrate your workloads to complete the update.
```

Comment:
Might be worth mentioning that the operator can also migrate the workloads for you if you specify `updateWorkloads=true`.

Reply:
That is not implemented though :)